toptal/gitignore.io
# updated templates from https://github.com/toptal/gitignore

# Description
This PR synchronizes templates from https://github.com/toptal/gitignore

https://github.com/toptal/gitignore/compare/aaefb4...a3f813d

# How to test

- `docker-compose up --build`
- go to http://localhost:8080
- type `Tye` into the search box and verify template exists
